src/ORM/Association/BelongsToMany.php

```diff
@@ -301,7 +302,7 @@ protected function _joinCondition($options)
      * @param \Cake\ORM\Query $fetchQuery The query to get results from
      * @param array $options The options passed to the eager loader
      * @return array
-     * @throws \RuntimeException when the association property is not part of the results set.
+     * @throws RuntimeException when the association property is not part of the results set.
```

Comment:
I think doc blocks need absolute class references to generate API docs properly.